stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/domain/message/internal/DraftMessageEntity.kt

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2014-2025 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.offline.repository.domain.message.internal
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(
+    tableName = DraftMessageEntity.DRAFT_MESSAGE_ENTITY_TABLE_NAME,
+)
+internal data class DraftMessageEntity(
+    @PrimaryKey
+    val id: String,
+    val cid: String,
+    val text: String,
+    val parentId: String? = null,
+    val mentionedUsersIds: List<String> = listOf(),
+    val silent: Boolean,
+    val showinChannel: Boolean,
+    val replyMessageId: String? = null,
+    val extraData: Map<String, Any> = mapOf(),
+) {
+    companion object {
+        internal const val DRAFT_MESSAGE_ENTITY_TABLE_NAME = "draft_message_entity"
```

Comment:
nit: Given that the constant is already part of `DraftMessageEntity`, the `DRAFT_MESSAGE_ENTITY_` prefix seems to be redundant.

`DraftMessageEntity.TABLE_NAME` would be clear enough.

Reply:
The name is a bit redundant, but as we are using it on the DAO query, it avoid us to use `{}` within the query string